src/test/resources/register.properties

```diff
@@ -0,0 +1,6 @@
+username = tester99
+email = tester99@mailinator.com
+password = 6vb8baGUXHRvB7s
+baseUrl = https://sandbox-api.violet.io/v1
+X-Violet-App-Secret = 60159de73a794d4089b0aabee2836891
+X-Violet-App-Id = 10062
```

Comment:
You don't need these. They are used in the API tests.

Reply:
I removed the specified lines.